CefSharp/Internals/JavascriptObjectRepository.cs

```diff
@@ -204,12 +210,19 @@ private void AnalyseObjectForBinding(JavascriptObject obj, bool analyseMethods,
                     continue;
                 }
 
-                var jsProperty = CreateJavaScriptProperty(propertyInfo);
+                var jsProperty = CreateJavaScriptProperty(propertyInfo, lowerCaseJavascriptNames);
                 if (jsProperty.IsComplexType)
                 {
                     var jsObject = CreateJavascriptObject();
                     jsObject.Name = propertyInfo.Name;
-                    jsObject.JavascriptName = LowercaseFirst(propertyInfo.Name);
+
+                    // depending on lowerCaseJavascriptNames JS Properties will be lowercase or unaltered 
+                    // defaults to LowercaseFirst, in order to not change existing behaviour
+                    if (lowerCaseJavascriptNames)
```

Comment:
If statements need braces. In this case, even for one liners.

In this case I think it's cleaner to pass in the param to `LowercaseFirst` and have it decide. (potentially rename `LowercaseFirst` to be more descriptive given it's additional responsibility).